contracts/WAXPERC20.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/Initializable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20Burnable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/access/Ownable.sol";
+
+/**
+ * @dev Vanilla upgradeable {ERC20} "WAXP" token:
+ *
+ */
+contract WAXPERC20UpgradeSafe is Initializable, OwnableUpgradeSafe, ERC20BurnableUpgradeSafe {
+    uint8 public constant DECIMALS = 8;                         // The number of decimals for display
+
+    /**
+     * See {ERC20-constructor}.
+     */
+
+    function initialize(address escrow) public initializer {
+        ERC20UpgradeSafe.__ERC20_init("WAXP Token", "WAXP");
+        _setupDecimals(DECIMALS);
+        uint256 INITIAL_SUPPLY = 386482894311326596;  // supply specified in base units
```

Comment:
@porkchop please verify that initialize value is correct. `INITIAL_SUPPLY` I get from [eosio.token stat table](https://wax.bloks.io/account/eosio.token?loadContract=true&tab=Tables&table=stat&account=eosio.token&scope=WAX&limit=100)

Reply:
That looks good @danielAlvess